discover/portal/articles/210-setting-up/02-instance-settings/00-intro.markdown

```diff
@@ -0,0 +1,24 @@
+# Setting Up a Virtual Instance [](id=setting-up-a-virtual-instance)
+
+Once @product@ is installed, the configuration begins. Whether that means adding
+another virtual instance or setting up the initial instance to suit your needs.
+Recall that configuration happens at different scopes. Here were thinking about
+the virtual instance scope. There's an important difference between the system
+scope and the instance scope. The system scope is the highest level scope you
+can make configurations at. All virtual instances are impacted by configuration
+done at this scope. The instance scope applies only to one particular virtual
+instance. 
+
+Virtual instances are separate Liferay installations (each with their own
```

Comment:
They are not separate installations, that's why they are virtual, because they share the same installation.

Reply:
Yeah, I see how this is incorrect. How can we describe how virtual instances differ to administrators? Is it really just the configuration scope? 
@JorgeFerrer

Reply:
Suggested correction:
>Virtual instances have unique domain names but share a server and database. Each
instance can be configured differently.
We talk more about this in the Virtual Instances article, so just a brief differentiator is good here. Let me know if this works.

Reply:
I think that correction is good. Maybe I would say a bit more than just "configured differently". Example:
Virtual instances have unique domain names but share a server and database. Each
virtual instance will have independent data and configuration.